spec/features/current_organization_spec.rb

```diff
@@ -38,11 +38,7 @@
       end
     end
 
-    it "see organization that was selected before sign out" do
```

Comment:
@shir Periodically failing spec was fixed early in other task. I want fix pending spec. What logic in this pending test? After sign out and sign in receive the same organization?